HomeFinder/Models/ListAttributesRepository.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HomeFinder.Models
+{
+    public class ListAttributesRepository //: IAttributesRepository
```

Comment:
if we are not using this class anywhere, we should delete it

Reply:
I will delete it